examples/Basic/Esp32DateTime.ino

```diff
@@ -0,0 +1,54 @@
+//#include <WiFi.h> // for WiFi shield Or Esp32
+//#include <WiFi101.h> // for WiFi 101 shield or MKR1000
+#include <ESP8266WiFi.h>
```

Comment:
The sketch name contains "ESP32", but this library is specific to the ESP8266 boards. The sketch won't compile for an ESP32 board.